src/mapkit.cs

```diff
@@ -1532,6 +1567,11 @@ partial interface MKMapCamera : NSCopying, NSSecureCoding {
 		[TV (13, 0), NoWatch, Mac (10, 15), iOS (13, 0)]
 		[Export ("centerCoordinateDistance")]
 		double CenterCoordinateDistance { get; set; }
+
+		[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, TV (16,0)]
+		[Static]
+		[Export ("cameraLookingAtMapItem:forViewSize:allowPitch:")]
+		MKMapCamera CameraLookingAtMapItem (MKMapItem mapItem, CGSize viewSize, bool allowPitch);
```

Comment:
```suggestion
		MKMapCamera CreateCameraLookingAt (MKMapItem MapItem, CGSize viewSize, bool allowPitch);
```

Reply:
@dalexsoto is there a special reason to capitalize the param name `MapItem`?

Reply:
no my bad, c&p error